### ISSUE TITLE
Can't start local Jets server without AWS credentials

### ISSUE THREAD

**Issue by onnimonni** [tongueroo/jets#60]
I don't usually have AWS access keys exposed as env or in `~/.aws/credentials` file in my default working environment and I doubt that many do.

I think the default Jets configuration should allow people to run local jets environment for testing locally without any dependency to AWS.

For example this is my Developer experience when I start using the Jets:
```bash
$ env
RUBYGEMS_VERSION=2.7.8
BUNDLER_VERSION=1.17.1
HOSTNAME=dcd54d225d69
OLDPWD=/app
RUBY_VERSION=2.5.3
GEM_HOME=/usr/local/bundle
PWD=/app/project
HOME=/root
BUNDLE_PATH=/usr/local/bundle
BUNDLE_APP_CONFIG=/usr/local/bundle
TERM=xterm
BUNDLE_SILENCE_ROOT_WARNING=1
RUBY_MAJOR=2.5
SHLVL=1
RUBY_DOWNLOAD_SHA256=1cc9d0359a8ea35fc6111ec830d12e60168f3b9b305a3c2578357d360fcf306f
PATH=/usr/local/bundle/bin:/usr/local/bundle/gems/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
_=/usr/bin/env
$ jets server
sh: 1: aws: not found
You can also get rid of this message by setting AWS_REGION or configuring ~/.aws/config with the region
Traceback (most recent call last):
	60: from /usr/local/bundle/bin/jets:23:in `<main>'
	59: from /usr/local/bundle/bin/jets:23:in `load'
	58: from /usr/local/bundle/gems/jets-1.0.9/exe/jets:14:in `<top (required)>'
	57: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/cli.rb:5:in `start'
	56: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/cli.rb:18:in `start'
	55: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/cli.rb:101:in `full_command'
	54: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/commands/base.rb:145:in `autocomplete'
	53: from /usr/local/bundle/gems/memoist-0.16.0/lib/memoist.rb:170:in `eager_load!'
	52: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/commands/base.rb:59:in `eager_load!'
	51: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/commands/base.rb:59:in `select'
	50: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/commands/base.rb:70:in `block in eager_load!'
	49: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/core_ext/string/inflections.rb:68:in `constantize'
	48: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/inflector/methods.rb:281:in `constantize'
	47: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/inflector/methods.rb:281:in `inject'
	46: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/inflector/methods.rb:281:in `each'
	45: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/inflector/methods.rb:285:in `block in constantize'
	44: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/inflector/methods.rb:285:in `const_get'
	43: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
	42: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
	41: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
	40: from /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	39: from /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	38: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/commands/deploy.rb:1:in `<top (required)>'
	37: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/commands/deploy.rb:2:in `<module:Commands>'
	36: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/commands/deploy.rb:5:in `<class:Deploy>'
	35: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
	34: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
	33: from /usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
	32: from /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	31: from /usr/local/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	30: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/timing.rb:13:in `<top (required)>'
	29: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/timing.rb:14:in `<module:Jets>'
	28: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/timing.rb:17:in `<module:Timing>'
	27: from /usr/local/bundle/gems/memoist-0.16.0/lib/memoist.rb:170:in `build_root'
	26: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/core.rb:53:in `build_root'
	25: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/core.rb:22:in `config'
	24: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/core.rb:15:in `application'
	23: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/application.rb:14:in `setup!'
	22: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/application.rb:81:in `load_configs'
	21: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/application.rb:121:in `set_aliases!'
	20: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/application.rb:125:in `set_iam_policy'
	19: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/application.rb:143:in `default_iam_policy'
	18: from /usr/local/bundle/gems/memoist-0.16.0/lib/memoist.rb:170:in `account'
	17: from /usr/local/bundle/gems/jets-1.0.9/lib/jets/aws_info.rb:51:in `account'
	16: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-sts/client.rb:1142:in `get_caller_identity'
	15: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/seahorse/client/request.rb:70:in `send_request'
	14: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/seahorse/client/plugins/response_target.rb:23:in `call'
	13: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/response_paging.rb:10:in `call'
	12: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/param_converter.rb:24:in `call'
	11: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/idempotency_token.rb:17:in `call'
	10: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/jsonvalue_converter.rb:20:in `call'
	 9: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/seahorse/client/plugins/raise_response_errors.rb:14:in `call'
	 8: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/param_validator.rb:24:in `call'
	 7: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/seahorse/client/plugins/endpoint.rb:45:in `call'
	 6: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/user_agent.rb:13:in `call'
	 5: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/query/handler.rb:28:in `call'
	 4: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/retry_errors.rb:150:in `call'
	 3: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/helpful_socket_errors.rb:10:in `call'
	 2: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/signature_v4.rb:65:in `call'
	 1: from /usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/signature_v4.rb:112:in `apply_signature'
/usr/local/bundle/gems/aws-sdk-core-3.36.0/lib/aws-sdk-core/plugins/signature_v4.rb:72:in `sign_request': unable to sign request without credentials set (Aws::Errors::MissingCredentialsError)
```

I noticed the `Jets::AwsInfo#test?` function and decided to run my local setup with `TEST=true`.

And immediately Jets started working as I was expecting:
```
$ export TEST=true
$ jets server
=> bundle exec shotgun --port 8888 --host 127.0.0.1
Jets booting up in development mode!
Jets::Rack#start
== Shotgun/WEBrick on http://127.0.0.1:8888/
[2018-11-05 18:04:32] INFO  WEBrick 1.4.2
[2018-11-05 18:04:32] INFO  ruby 2.5.3 (2018-10-18) [x86_64-linux]
[2018-11-05 18:04:32] INFO  WEBrick::HTTPServer#start: pid=474 port=8888
```

**Comment by tongueroo** [comment]
Yup 👍 Adding to todo. Thanks again.

**Comment by tongueroo** [comment]
Thanks again for the fix! 🎉 Merged and released in 1.0.11 [Changelog](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md)
